src/vfs/public.c

```diff
@@ -152,6 +161,36 @@ static bool vfs_mkparents_recurse(const char *path) {
 
 	return vfs_mkparents_recurse(p) && vfs_mkdir(p);
 }
+*/
+static bool vfs_mkparents_recurse(const char *path) {
+
+    char p[strlen(path) + 1];
+    VFSInfo i;
+    strcpy(p,path);
```

Comment:
Since the length of the string is known, just use `memcpy`:

```suggestion
    memcpy(p, path, sizeof(p));
```